src/Component/Footer/Footer.jsx

```diff
@@ -1,47 +1,77 @@
-import React, { useState, useEffect } from 'react';
-import '../../CSS/Footer.css'
-import SearchBox from './SearchBox'
-import 'bootstrap/dist/css/bootstrap.min.css';
-
+import React, { useState, useEffect } from "react";
+import "../../CSS/Footer.css";
+import SearchBox from "./SearchBox";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { Link } from "react-router-dom";
 function Footer() {
-    const navStyle = {
-        color: "white"
-    }
-
-    
-    useEffect(() => {
-        fetchItems();
-    }, []);
-    
-    const [items, setItems] = useState([]);
-    const fetchItems = async() =>{

```

Comment:
@Beloved1310 

Create a file for  your constants and  add your base api url to  the file then export and use anywhere in your application